modules/javafx.web/src/test/resources/test/html/cookie.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="Refresh" content="100; URL=ipsum.html" />
```

Comment:
The redirect to another html file seems unnecessary. Is there a reason you added it? If I locally remove the `<head>...</head>` block the still passes.